### PR TITLE
apps/projects/participants: send welcome email when participant is ad…

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
…ded to project

I still need to add a test, but you could already try it out to merge when that is done. :)

### CODE REVIEW

File: apps/projects/templates/a4_candy_projects/emails/welcome_participant.en.email [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends 'email_base.'|add:part_type %}
+{% load i18n %}
+
+{% block subject %}{% if project.is_semipublic %}{% blocktrans with project_name=project.name %}Link to semi-public project: "{{ project_name }}"{% endblocktrans %}{% else %}{% blocktrans with project_name=project.name %}Link to private project: "{{ project_name }}"{% endblocktrans %}{% endif %}{% endblock %}
+
+{% block headline %}{% if project.is_semipublic %}{% blocktrans with project_name=project.name %}Link to semi-public project: "{{ project_name }}"{% endblocktrans %}{% else %}{% blocktrans with project_name=project.name %}Link to private project: "{{ project_name }}"{% endblocktrans %}{% endif %}{% endblock  %}
+
+{% block content %}{% if project.is_semipublic %}{% blocktrans with project_name=project.name %}you have accepted the invitation to the semi public project "{{ project_name }}". To participate click on "Go to Project".{% endblocktrans %}{% else %}{% blocktrans with project_name=project.name %}you have accepted the invitation to the private project "{{ project_name }}". To participate click on "Go to Project".{% endblocktrans %}{% endif %}{% endblock %}
```
Comment (Rineee) [inline apps/projects/templates/a4_candy_projects/emails/welcome_participant.en.email:8]: The dash in semi-public is missing here, just to be consistent :nerd_face:

**Review by Rineee — APPROVED** [review]
Yes, this works perfectly! In the e-mail at one point there it says semi public instead of semi-public.. but apart from that approved and you can merge when test is there!

**Comment by fuzzylogic2000** [comment]
Compared to master the coverage increased 0.54!!

**Comment by fuzzylogic2000** [comment]
@Rineee I will merge, you already said it's fine!

**Pull request merged**